### PR TITLE
new version README template

### PR DESCRIPTION
**Pull request by juanmaguitar**
UPDATE template for individual READMEs
This version simplifies the instructions and put at the same level the two main use cases to check the examples locally:
- as part of the monorepo
- as part of the user's own WordPress installation

If approved, we'd need to update the individual READMEs using this template

### CODE REVIEW

File: templates/block-examples/readmes/README-EXAMPLE.tpl.md [hunk 1]
```diff
@@ -5,31 +5,41 @@
 
 ## Start Guide
 
-#### 1. Install dependencies
+The examples in this repo require these three steps to see them in action:
 
-Check the [Repo Commands > Dependencies](../../DEVELOPMENT.md#dependencies) instructions for this repo
+1. Install dependencies
+2. Generate the build
+3. Use it in a WordPress
```
Comment (ryanwelcher) [inline templates/block-examples/readmes/README-EXAMPLE.tpl.md]:
```suggestion
3. Use it in a WordPress installation
```

Comment (ryanwelcher) [reply templates/block-examples/readmes/README-EXAMPLE.tpl.md]: I think we're missing a word.

File: templates/block-examples/readmes/README-EXAMPLE.tpl.md [hunk 2]
```diff
@@ -5,31 +5,41 @@
...
-<p>If you do that, you'll need to do the following</p>
-<ul>
-<li>Remove any <code>node_modules</code> folder inside this folder</li>
-<li>Run <code>npm install</code> to install the dependencies</li>
-<li>Run <code>npm build</code> to generate the "build" version of the blocks</li>
-<li>Activate the plugin in your own WordPress installation</li>
-<ul>
-</details>
-<br>  
+The quickest way to have access to all the examples is clone this repository and check the examples locally.
```
Comment (ryanwelcher) [inline templates/block-examples/readmes/README-EXAMPLE.tpl.md]:
```suggestion
The quickest way to have access to all the examples is clone this repository and check out the examples locally.
```

File: templates/block-examples/readmes/README-EXAMPLE.tpl.md [hunk 3]
```diff
@@ -5,31 +5,41 @@
...
+
+Check the [05 Development > Repo Commands > Dependencies](https://github.com/wptrainingteam/block-development-examples/wiki/05-Development#dependencies) instructions for this repo to install the dependencies of this example.
+
+Once the depencies are installed (a `node_modules` folder should have been created), [launch the build process](https://github.com/wptrainingteam/block-development-examples/wiki/05-Development#build-process) to get the final version of the block that can be used from WordPress.
```
Comment (ryanwelcher) [inline templates/block-examples/readmes/README-EXAMPLE.tpl.md]:
```suggestion
Once the depencies are installed (a `node_modules` folder should have been created), [launch the build process](https://github.com/wptrainingteam/block-development-examples/wiki/05-Development#build-process) to get the final version of the block that can be used in WordPress. 
```

File: templates/block-examples/readmes/README-EXAMPLE.tpl.md [hunk 4]
```diff
@@ -5,31 +5,41 @@
...
+
+Check the [05 Development > Repo Commands > Dependencies](https://github.com/wptrainingteam/block-development-examples/wiki/05-Development#dependencies) instructions for this repo to install the dependencies of this example.
+
+Once the depencies are installed (a `node_modules` folder should have been created), [launch the build process](https://github.com/wptrainingteam/block-development-examples/wiki/05-Development#build-process) to get the final version of the block that can be used from WordPress. 
+
+Check the [WordPress Local Development Environment](https://github.com/wptrainingteam/block-development-examples/wiki/05-Development#wordpress-local-development-environment) to learn how to quickly get your local WordPress installation to see this example.
+
+### As an isolated example on your own WordPress installation
+
+If you have a local WordPress installation already running, you can also put the plugin folders of the examples you're interested in (by copyin and paste, from the zips...) inside the `plugins` folder of that installation and check the examples there.
```
Comment (ryanwelcher) [inline templates/block-examples/readmes/README-EXAMPLE.tpl.md]:
```suggestion
If you have a local WordPress installation already running, you can also put the plugin folders of the examples you're interested in (by copying and pasting, from the zips...) inside the `plugins` folder of that installation and check the examples there.
```

**Review by ryanwelcher — CHANGES_REQUESTED** [review]
Some minor changes in the readme. Do we also need to move this into the readme that is [part of the template?](https://github.com/wptrainingteam/block-development-examples/blob/trunk/templates/block-examples/templates/plugin/README.md.mustache)

**Comment by juanmaguitar** [comment]
@ryanwelcher suggestions accepted 👍

> Do we also need to move this into the readme that is [part of the template?](https://github.com/wptrainingteam/block-development-examples/blob/trunk/templates/block-examples/templates/plugin/README.md.mustache)

Yes! We would need to update all the individual readme's and the template

**Review by ryanwelcher — APPROVED** [review]
Looks good to me. Do we want to tackle the tempate/existing readme files in a separate PR?

**Comment by juanmaguitar** [comment]
> Looks good to me. Do we want to tackle the tempate/existing readme files in a separate PR?

Yes, I'll open a separate PR for those changes

**Pull request merged**